parse_emails/tests/test_data/chinese.eml

```diff
@@ -0,0 +1,9 @@
+Content-Type: text/plain; charset="gb2312"
+MIME-Version: 1.0
+Content-Transfer-Encoding: base64
+Subject: Greeting in Chinese
+From: Sender Name <sender@example.com>
+To: Recipient Name <recipient@example.com>
+Date: Mon, 06 Mar 2023 19:47:06 +0200
```

Comment:
shouldnt there be chinese chars in this test?

Reply:
its encoded in base64, so no.

Reply:
@JudahSchwartz ?